src/pages/RepoPage/BundlesTab/BundleOnboarding/BundleOnboarding.tsx

```diff
@@ -26,30 +28,102 @@ const Loader = () => (
   </div>
 )
 
+const BUNDLER_OPTIONS = {
+  Vite: 'Vite',
+  Rollup: 'Rollup',
+  Webpack: 'Webpack',
+} as const
+type BundlerOption = keyof typeof BUNDLER_OPTIONS
+type BundlerOptionUrls = Record<BundlerOption, string>
+
+const getInitialBundler = (path: string, urls: BundlerOptionUrls) => {
+  if (path === urls.Vite) {
+    return BUNDLER_OPTIONS.Vite
+  } else if (path === urls.Rollup) {
+    return BUNDLER_OPTIONS.Rollup
+  } else if (path === urls.Webpack) {
+    return BUNDLER_OPTIONS.Webpack
+  }
+}
+
+interface BundlerSelectorProps {
+  provider: string
+  owner: string
+  repo: string
+}
+
+function BundlerSelector({ provider, owner, repo }: BundlerSelectorProps) {
+  const location = useLocation()
+  const history = useHistory()
+  const { bundleOnboarding, bundleWebpackOnboarding, bundleRollupOnboarding } =
+    useNavLinks()
+  const urls = useMemo(
+    () => ({
+      Vite: bundleOnboarding.path({ provider, owner, repo }),
+      Rollup: bundleRollupOnboarding.path({ provider, owner, repo }),
+      Webpack: bundleWebpackOnboarding.path({ provider, owner, repo }),
+    }),
+    [
+      bundleOnboarding,
+      bundleWebpackOnboarding,
+      bundleRollupOnboarding,
+      provider,
+      owner,
+      repo,
+    ]
+  )
```

Comment:
We can probably skip memoizing this, it's not really a complex enough function that we should consider memoizing for